src/schema/v2/conversation/conversations.ts

```diff
@@ -18,22 +48,65 @@ const Conversations: GraphQLFieldConfig<void, ResolverContext> = {
     },
   }).connectionType,
   description: "Conversations, usually between a user and partner.",
-  args: pageable(),
-  resolve: (_root, options, { conversationsLoader, userID }) => {
-    if (!conversationsLoader) return null
-    const { page, size, offset } = convertConnectionArgsToGravityArgs(options)
+  args: pageable({
+    dismissed: {
+      type: GraphQLBoolean,
+    },
+    hasMessage: {
+      type: GraphQLBoolean,
+    },
+    hasReply: {
+      type: GraphQLBoolean,
+    },
+    partnerId: {
+      type: GraphQLString,
+    },
+    type: {
+      type: ConversationsTypeEnum,
+    },
+  }),
+  resolve: (_root, args, { conversationsLoader, userID }) => {
+    if (!conversationsLoader) {
+      return null
+    }
+
+    const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
     const expand = ["total_unread_count"]
+
+    let params
+    if (args.type === "Partner") {
+      if (!args.partnerId) {
+        throw new Error("Argument `partnerId` is required.")
+      }
+
+      params = {
+        deleted: false,
+        intercepted: false,
+        to_id: args.partnerId,
+        to_type: "Partner",
+
+        // TODO
+        dismissed: args.dismissed,
+        has_message: args.hasMessage,
+        has_reply: args.hasReply,
```

Comment:
Marked these as `TODO` in the sense that I've yet to work with them directly and will likely have questions about their use

Reply:
I'll also take a look at those on Impulse and Volt, so I can add this to our docs.

On a quick glance here, it looks like we really want these three optional args to be flexible according to CMS requests (depending on the conversation tab being accessed, on the filters being used and also for new inquiries notifications). But it's definitely worth digging a bit deeper into it, thanks for the heads up.

Reply:
Yup, we can do whatever here! (And they're currently optional, so np)

Reply:
#superminor i _might_ leave these out for now in that case. It's so easy to add in when actually known/needed, can't help but feel it's premature otherwise

Reply:
We're going to be using these very soon! So will keep around and let them fall back to undefined